frontend/app/services/activity-service.js

```diff
@@ -51,6 +55,16 @@ module.exports = function(HALAPIResource, $http, PathHelper){
       };
 
       return activity.links.update.fetch(options).then(function(activity){
+        notification = NotificationsService.addSuccess(
+          I18n.t('js.work_packages.comment_updated')
+        );
+        // Remove the notification some time later
+        // but only when we're not in accessible mode
+        if (!ConfigurationService.accessibilityModeEnabled()) {
+          $timeout(function() {
+            NotificationsService.remove(notification);
```

Comment:
Removed variable, as we're not addressing the `hide after timeout` any longer in here.